### PR TITLE
Update to v0.15.0

### PR DESCRIPTION
**Pull request by JordanMartinez**
**Description of the change**

Backlinking to purescript/purescript#4244

Updates project to compile on v0.15.0 PureScript.

---

**Checklist:**

- [x] Added the change to the changelog's "Unreleased" section with a reference to this PR (e.g. "- Made a change (#0000)")
- [x] Linked any existing issues or proposals that this pull request should close
- [ ] Updated or added relevant documentation
- [ ] Added a test for the contribution (if applicable)

### CODE REVIEW

**Comment by JordanMartinez** [comment]
:ping_pong: @thomashoneyman

File: bower.json [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+  },
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "output"
+  ],
+  "dependencies": {
+    "purescript-psci-support": "master"
+  }
```
Comment (thomashoneyman) [inline bower.json]: This should be a `devDependency`. And with that, this isn't even a breaking change!

**Comment by JordanMartinez** [comment]
How is this not a breaking change?

**Comment by JordanMartinez** [comment]
Ah, nevermind.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,7 @@ Notable changes to this project are documented in this file. The format is based
 ## [Unreleased]
 
 Breaking changes:
+- Update project and deps to PureScript v0.15.0 (#26 by @JordanMartinez)
```
Comment (thomashoneyman) [inline CHANGELOG.md]: We could move this to 'Other improvements' and just say "Updated CI to track unstable PureScript" and release this version as 3.0.1

Comment (JordanMartinez) [reply CHANGELOG.md]: I did something like that.

**Review by thomashoneyman — APPROVED** [review]

**Pull request merged**